controllers/nodehealthcheck_controller.go

```diff
@@ -539,14 +595,11 @@ func (r *NodeHealthCheckReconciler) remediate(node *v1.Node, nhc *remediationv1a
 	if err != nil {
 		//An unhealthy node exist but remediation couldn't be created because lease wasn't obtained - update unhealthy nodes and requeue
 		if _, isLeaseAlreadyTaken := err.(lease.AlreadyHeldError); isLeaseAlreadyTaken {
-			resources.UpdateStatusNodeUnhealthy(node, nhc)
```

Comment:
NIT: If we don't update then the above comment should mention an update.

Reply:
done